app/lib/three_scale/open_api/url_resolver.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'addressable/template'
+
+module ThreeScale
+  module OpenApi
+    class UrlResolver
```

Comment:
```suggestion
    class Parser
```

wdyt?

Reply:
No a big deal for me but imo `Parser` or `Resolver` is better because being inside `ThreeScale::OpenApi` it is already implied that we are talking about a URL :smile:

Reply:
Good :+1: Makes sense :+1: :muscle: